miden/src/cli/debug/command.rs

```diff
@@ -9,7 +9,7 @@ pub enum DebugCommand {
     PrintStack,
     PrintStackItem(usize),
     PrintMem,
-    PrintMemAddress(u64),
+    PrintMemAddress(u32),
```

Comment:
Let's use `MemoryAddress` here instead of `u32`

Reply:
Done (this is the reason for deriving `Hash` for `MemoryAddress`).